ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml

```diff
@@ -28,94 +30,83 @@ StatusDialog {
             StatusButton {
                 objectName: "getStartedStatusButton"
                 enabled: acknowledge.checked && termsOfUse.checked
-                size: StatusBaseButton.Size.Large
-                font.weight: Font.Medium
-                text: qsTr("Get Started")
+                text: qsTr("Get started")
                 onClicked: root.close()
             }
         }
     }
 
-    contentItem: Item {
-        Column {
-            width: 416
-            spacing: Style.current.padding
-            anchors.centerIn: parent
+    contentItem: ColumnLayout {
+        spacing: Style.current.padding
+
+        StatusCheckBox {
+            Layout.fillWidth: true
+            id: acknowledge
```

Comment:
neat: I prefer to have `id` as the first property always, then `objectName` than others (the same for below, but didn't say you need to fix, up to you ;) ).

Reply:
Yeah... I guess everybody likes it different; for me the attached properties always come before the own ones :)